README.md

```diff
@@ -520,7 +520,6 @@ It is available in [Maven central](https://search.maven.org/#search%7Cgav%7C1%7C
 # Future improvements
 
 * Offer the possibility to generate runners and features directly in the target folder
-* Integrate Travis CI.
```

Comment:
Full Travis integration is work in progress. At the moment it just checks if the build is ok.